modules/sql/servers/elastic-pools/README.md

```diff
@@ -41,7 +41,7 @@ This module deploys an Azure SQL Server Elastic Pool.
 | `location` | string | `[resourceGroup().location]` |  | Location for all resources. |
 | `maintenanceConfigurationId` | string | `''` |  | Maintenance configuration resource ID assigned to the elastic pool. This configuration defines the period when the maintenance updates will will occur. |
 | `maxSizeBytes` | int | `34359738368` |  | The storage limit for the database elastic pool in bytes. |
-| `minCapacity` | int | `2` |  | Minimal capacity that serverless pool will not shrink below, if not paused. |
+| `minCapacity` | int | `-1` |  | Minimal capacity that serverless pool will not shrink below, if not paused. |
```

Comment:
Is there any documentation around this value and how it is intended to be used that we may link here?

Reply:
The default value comes from what exported when deployed manually through the portal. Missing detailed official documentation about this value and proper usage